diff --git a/node_modules/react-native-pager-view/android/build.gradle b/node_modules/react-native-pager-view/android/build.gradle
index b46c366..f3c3765 100644
--- a/node_modules/react-native-pager-view/android/build.gradle
+++ b/node_modules/react-native-pager-view/android/build.gradle
@@ -4,11 +4,11 @@ buildscript {
 
   repositories {
     google()
+    maven { url 'https://repo1.maven.org/maven2/' }
     mavenCentral()
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:4.2.1'
     // noinspection DifferentKotlinGradleVersion
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
   }
